### PR TITLE
Fixing doc block issues

### PR DESCRIPTION
**Pull request by dhensby**
Continuation of #4274 because "oh well" isn't good enough :P

### CODE REVIEW

File: forms/Form.php [hunk 1]
```diff
@@ -546,9 +546,8 @@ public function makeReadonly() {
 	 * form on the page upon validation errors in the form or if
 	 * they just need to redirect back to the page
 	 *
-	 * @param bool $bool
+	 * @param bool $bool Redirect to form on error?
```
Comment (chillu) [inline forms/Form.php]: Why is this a question?

Comment (dhensby) [inline forms/Form.php]: The bool answers the Q...

Comment (dhensby) [inline forms/Form.php]: But, yes, is a bit odd... I could remove the `?` ?

**Comment by chillu** [comment]
All good, I think it was just a case of still being half asleep when looking at it ;)

**Pull request merged**